src/Views/Form/Themes/Sequoia/Actions.php

```diff
@@ -14,6 +14,8 @@
  */
 class Actions {
 
+	protected $theme_options;
```

Comment:
@henryholtgeerts in `src` folder for `psr-4` coding standard, so property name must be in camel case

Reply:
fixed: https://github.com/impress-org/givewp/compare/b1375fdefbcc...6d7c638d8365